cogs/roles.py

```diff
@@ -141,6 +141,32 @@ async def inrole(self, ctx, *, queryRole):
                           editable_content=False)
             await pages.paginate()
 
+    @commands.command()
+    async def inchannel(self, ctx):
+        """Returns list of users in current channel"""
+        members = None
+        channel = ctx.message.channel
+        members = channel.members
+
+        if members is None:
+            return
+
+        channelUsers = list(map(lambda m: str(m) + "\n", members))
+        header = f"List of users in {channel.mention} - {len(members)}"
+
+        # TODO remove for paginator take empty list for embed
+        if len(channelUsers) == 0:
```

Comment:
```python
if not channel_users:
```
is cleaner

Reply:
this isn't possible anyway, see my other review but there's always at least the bot and the person calling the command

Reply:
Resolved in b20d0ac